src/Files/UserControls/RestartControl.xaml

```diff
@@ -35,7 +36,7 @@
                         <Button
                             x:Name="YesButton"
                             x:Uid="ButtonYes"
-                            Background="{StaticResource SystemAccentColor}"
+                            Style="{ThemeResource AccentButtonStyle}"
```

Comment:
Is there any reason to change the full style?

Reply:
This is because the hover style does not apply.

Reply:
If you don't need it, you can delete it.